I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 34, in script
    jump scenario_1
ScriptError: could not find label 'scenario_1'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 34, in script
    jump scenario_1
  File "renpy/ast.py", line 1752, in execute
    rv = renpy.game.script.lookup(target)
  File "renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'scenario_1'.

Windows-10-10.0.19041
Ren'Py 7.4.8.1895
TimeyWimey 1.0
Tue Nov 16 11:32:53 2021
